eve/main.yml

```diff
@@ -325,6 +325,8 @@ models:
         FINAL_STATUS: ''
         TEXT: |-
           Artifact Url: %(prop:artifacts_public_url)s
+          Branch: %(prop:branch)s
+          Commit: %(prop:revision)s
```

Comment:
Cool ! Could we also add a link to the build?

```
Build URL: https://eve.devsca.com/github/scality/metalk8s/#/builders/1/builds/%(prop:buildnumber)s
```

Reply:
That do not work the `1` builder is not always 1 and this information is currently not in eve property a ticket is in progress about this on there side see #2371, once this is done I will add it